.devcontainer/devcontainer.json

```diff
@@ -19,7 +19,9 @@
     // Add the IDs of extensions you want installed when the container is created.
     "extensions": [
         "ms-python.python",
-        "ms-python.vscode-pylance"
+        "ms-python.vscode-pylance",
+        "ryanluker.vscode-coverage-gutters",
+        "tamasfe.even-better-toml"
```

Comment:
The `.vscode/extensions` are recommendations, so I'm happy to put extras in there, but these add to the container build time so I'd prefer to only install pylance here.

Reply:
Just tried re-building without cache and they seem to add ~1s to the build time. With toml as a core language in the project I think it makes sense to build the container with a language server for it, happy to drop coverage gutters to a recommendation though

Reply:
I would argue yes to all of these, sphinx more than yaml. That being said this is probably best discussed in it's own issue. So I shall remove them from the dev container for this PR. Settings sync was an absolute pain for me, as I frequently switch between python, C, Rust & CUDA codes.